README.md

```diff
@@ -19,28 +19,8 @@ in the test suite.
 
 ## Benchmarking
 
-A simple benchmark is included in the `benchmark` directory. This script runs
-a simple program including repetition and records on an input of 1000000
-items. The directory also includes scripts for running the benchmark in
-node.js, the SpiderMonkey shell and ChakraCore. Make sure these are
-installed, open a shell in the benchmark directory and then run the
-`run-all.sh` script. Alternatively, use one of the following, depending on
-the engine to test on:
-
-### Node.js
-
```

Comment:
This all is no longer applicable?

Reply:
Correct, we now allow benchmarks to be run in node.js or in a browser.

Reply:
Jos and I discussed this earlier - it seems that the scripts did not age well and Jos was no longer able to get some of the old tests to run. This new Benchmark.js based approach should hopefully last longer.